packages/babel-helper-remap-async-to-generator/src/index.js

```diff
@@ -14,11 +14,24 @@ let buildWrapper = template(`
   })
 `);
 
+let arrowBuildWrapper =  template(`
+  (() => {
+    var ref = FUNCTION;
+    return (PARAMS) => ref.apply(this, arguments);
```

Comment:
side note: why do i only ever see expected/actual tests added and rarely see tests where the code is actually executed? actually running the code would catch a lot of these errors, even if it's only run on the latest version of node.